PackageInfo.g

```diff
@@ -112,11 +112,20 @@ Dependencies := rec(
 
 AvailabilityTest := function()
   local dir, str;
-  dir := DirectoriesPackagePrograms("curlInterface");
-  if Filename(dir, "curl.so") = fail then
-    str := "kernel extension not compiled, package cannot be loaded";
-    LogPackageLoadingMessage(PACKAGE_WARNING, str);
-    return fail;
+  if CompareVersionNumbers(GAPInfo.Version, "4.12") then
```

Comment:
Do we really gain much by supporting older GAP versions at this point? How many people will try to install a newer version of this package while insisting on staying on an older GAP version?

Reply:
Good point, as discussed updating to remove these cases.